README.md

```diff
@@ -1,6 +1,35 @@
+<!-- Badge for License -->
+<div align="right">
+
+  [![](https://img.shields.io/github/license/romainl/idiomatic-vimrc.svg?style=flat-square)](./LICENSE)
+
+</div>
+
+<!-- Logo -->
+<p align="center">
+  <a href="https://github.com/romainl/idiomatic-vimrc">
+    <img src="https://i.imgur.com/C6mxszA.png" alt="Idiomatic VIMRC" height="150px" />
+  </a>
+</p>
+
+<!-- Title and Description -->
+<div align="center">
+
 # IDIOMATIC VIMRC
 
-Guidelines for sculpting your very own `~/.vimrc`.
+🔨 _Guidelines for sculpting your very own `~/.vimrc`_
+
+</div>
+
+<div align="center">
```

Comment:
I feel that the HTML is unnecessary. The text below should be enough:

    This document is available in the following languages:

    - [English](./README.md)
    - [Traditional Chinese](./README-zh_TW.md)